Gopkg.lock

```diff
@@ -187,7 +243,10 @@
 
 [[projects]]
   name = "github.com/prometheus/client_golang"
-  packages = ["prometheus"]
+  packages = [
+    "prometheus",
+    "prometheus/promhttp"
```

Comment:
not sure if this is needed for rs/cors

Reply:
I didn't add that, `make build` did. Do I need to explicitly ignore that?